### PR TITLE
RGO2019

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
fixes https://github.com/tue-robotics/tue_robocup/issues/800

Merge together with https://github.com/tue-robotics/hero_bringup/pull/16, tue-robotics/robot_launch_files#45 and tue-robotics/people_detector#7

### CODE REVIEW

**Comment by ar13pit** [comment]
Do not merge this before robot_launch_files and people_detector PRs

File: ed_people_detector/src/people_detector_plugin.cpp [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+    }
+
+    ROS_DEBUG_STREAM("[ED People Detector]: received detection of " << res_3d.people.size() << " people");
+
+    for (auto it = world_->begin(); it != world_->end(); ++it)
+    {
+        const ed::EntityConstPtr& e = *it;
+        if (e)
+        {
+            if (e->hasType("unknown_person"))
```
[the remark this answers is not in the document]
Comment (LoyVanBeek) [reply ed_people_detector/src/people_detector_plugin.cpp]: We can change this API later on though

**Comment by ar13pit** [comment]
@LoyVanBeek if you have no comments then I'm good with this.

**Comment by ar13pit** [comment]
@LoyVanBeek please open an issue with all your comments about enhancements. 

I think this PR is RTM.

**Comment by MatthijsBurgh** [comment]
I will merge this one. We can fix stuff in other PRs if behaviour is undesired.

**Pull request merged**